src/main/java/brandkon/brand/Brand.java

```diff
@@ -0,0 +1,46 @@
+package brandkon.brand;
+
+import brandkon.category.Category;
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+public class Brand {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String imageUrl;
+    private String guidelines;
+
+    @OneToMany(mappedBy = "brand")
+    private List<BrandCategory> brandCategory;
```

Comment:
```suggestion
    private List<BrandCategory> categories;
```
이름은 이런 식으로 복수형으로 짓는 것이 더 좋을 것 같아요